README.md

```diff
@@ -1,220 +1,172 @@
-![logo](https://owo.whats-th.is/8mT5kxc.png)
-##### Project rewrite coming soon™
-[![Build Status](https://travis-ci.com/Pierce01/MinecraftLauncher-core.svg?branch=master)](https://travis-ci.com/Pierce01/MinecraftLauncher-core)
```

Comment:
If you want to keep the build badge, you can probably use the ones from GitHub action (if you configured something)